services/cd-service/pkg/cmd/server.go

```diff
@@ -91,6 +91,11 @@ func RunServer() {
 			defer tracer.Stop()
 		}
 
+		span, ok := tracer.SpanFromContext(ctx)
```

Comment:
is it ok to run this when `EnableTracing=false`?

Reply:
According to the documentation: `If the tracer is not started, calling this function is a no-op.`